tests/test_eflyt/test_case.py

```diff
@@ -0,0 +1,55 @@
+'''Tests related to the Case module'''
+
+import os
+import unittest
+
+from dotenv import load_dotenv
+from selenium.webdriver.common.by import By
+
+from itk_dev_shared_components.eflyt.eflyt_login import login
+from itk_dev_shared_components.eflyt.eflyt_search import open_case
+from itk_dev_shared_components.eflyt import eflyt_case
+
+load_dotenv()
+test_cpr = os.getenv("TEST_CPR")
+test_case = os.getenv("TEST_CASE")
+
+
+class CaseTest(unittest.TestCase):
+    """Test the Case functionality of Eflyt integration"""
+
+    @classmethod
+    def setUpClass(cls):
+        """Setup test class"""
+        eflyt_credentials = os.getenv("EFLYT_LOGIN").split(",")
+        cls.browser = login(eflyt_credentials[0], eflyt_credentials[1])
+
+    def setUp(self):
+        """Go to a clean case state"""
+        open_case(self.browser, test_case)
+
+    def test_get_beboere(self):
+        """Test inhabitant functions"""
+        inhabitants = eflyt_case.get_beboere(self.browser)
+        self.assertGreater(len(inhabitants), 0, "No inhabitants found on address")
```

Comment:
Cases very often have 0 inhabitants.
Also maybe dig into the inhabitant object e.g. self.assertRegex(inhabintants[0].cpr, "\d{6}-\d{4}")